### ISSUE TITLE
Brainstorming about tools for (Brainstorming about (x))

### ISSUE THREAD

**Issue by virgiliolino** [humanetes-com/proof-of-positive-interaction#18]
github issues are perhaps the reason in so many years I couldn't enter into the opensource communities...it become a set of parchment you have to unroll to get some kind of metal model...
In the last years I started to finally understand what is web2. Sometimes I had calls with people, opened collaboratively a spreadsheet, and everyone has a color, is allowed to contributes on a cell, I'm not talking about the need for synchronicity, interaction could be asynchronous, but everyone is allowed to interact in two dimensions.
lets not talk for now about web3...this is beyond. With this issue I'd like to suggest for defining concepts, brainstorming a different tool. I will try something like miro, if someone knows about some opensource equivalent that is usable great. Else I'll define a few use cases in miro, prepare some eventstorming sessions.
Please give me your opinion @Cooksey99 @DeluxeRaph @bsandipan88

**Comment by virgiliolino** [comment]
that's an amazing tool! We should use it

**Comment by Cooksey99** [comment]
I'm in favor of AppFlowy. It's built using Flutter and Rust. Flutter is perfect for this type of application, as we know it can be supported on any device (I'm a fan of Flutter). The other application suggested looks to only support Mac. Also, since I know the technologies used with AppFlowy and it's open source, that opens more possibilities in the future.

**Comment by virgiliolino** [comment]
I agree with @Cooksey99 and @bsandipan88 in favor of using AppFlowy. @DeluxeRaph anything against? else lets use AppFlowy
